packages/generators/generators/lib/plops/content-type.js

```diff
@@ -80,7 +81,7 @@ module.exports = plop => {
         return Object.assign(object, { [answer.attributeName]: val }, {});
       }, {});
 
-      const filePath = getFilePath(answers.destination);
+      const filePath = resolveFilePath(answers.plugin,getFilePath(answers.destination));
```

Comment:
You can, instead, update the `getFilePath` method to add the `/server` suffix to the plugin path:

https://github.com/strapi/strapi/blob/3706033d9fbf994376f31567d3adb659779854fd/packages/generators/generators/lib/plops/utils/get-file-path.js#L9

Reply:
done